### ISSUE TITLE
LaTeX does not recognize command definition after global search & replace

### ISSUE THREAD

**Issue by henrikgit** [texstudio-org/texstudio#457]
### Environment
- TeXstudio: 2.12.14
- Qt: 5.12.0, kompiliert mit Qt 5.11.2 R
- OS: Arch Linux 4.20
- TeX distribution: Tex Live

### Expected behavior
Commands get recognized after I replaced them with the enhanced/global search & replace function.

### Actual behavior
After using the enhanced search, latex throws error for commands which I've just replaced with the s&r function. The code itself is new and reads like it should be, i.e. they all were replaced correctly. 

However latex does not seem to reload the preamble... I think. 
TXS still outputs a latex error which says the old command (which I've just replaced) is an undefined control sequence.

When I click on it, it goes to the location of that error and TXS shows me a latex file (used via \input). But this file contains the new command, as intended. So while LaTeX says the old command is undefined (which is true), my latex document shows the new command, which is defined in the preamble.

### How to reproduce
1. Define a command in the preamble.
2. Create some content in latex files and use them via \input.
3. Use the enhanced s&r.
4. Run latex.

**Comment by sunderme** [comment]
are you sure that the file is saved ?
latex uses only the saved files on the disk, not the texts in the editor

**Comment by henrikgit** [comment]
No. And that question led me to the core issue perhaps, and a workaround, which is: 

* TXS  does not seem to open all the files it finds the search term in. Hence they, well, are not open and do not have that disk symbol which symbolizes unsaved changes.

* I have to click on a single error msg for each file the error in my project the error is output for so that the file is opened. Upon opening, they then get the disk icon.

And finally, pressing F5/running latex goes through smoothly.
